tasks/42_first-n-smallest/first-n-smallest.js

```diff
@@ -0,0 +1,14 @@
+function firstNSmallest(array, n) {
+  const result = [];
+  const sortedArr = array.slice().sort((a, b) => a - b);
+  const smallestNumbers = sortedArr.slice(0, n);
+  array.forEach((num) => {
+    if (smallestNumbers.includes(num)) {
```

Comment:
Looks like you can use `filter()` here instead.

Reply:
I even got rid of array initialization this way, thanks)